types/msg.go

```diff
@@ -77,8 +79,41 @@ type BankMsg struct {
 // SendMsg contains instructions for a Cosmos-SDK/SendMsg
 // It has a fixed interface here and should be converted into the proper SDK format before dispatching
 type SendMsg struct {
-	ToAddress   string `json:"to_address"`
-	Amount      Coins  `json:"amount"`
+	ToAddress string `json:"to_address"`
+	Amount    Coins  `json:"amount"`
+}
+
+type IbcMsg struct {
+	Ics20Transfer *Ics20TransferMsg `json:"ics20_transfer,omitempty"`
```

Comment:
Just curious, do you expect other IBC transfer types coming?

Reply:
The name `Ics20Transfer` was my idea originally. Now that I observed everyone is referring to it as [MsgTransfer](https://github.com/cosmos/cosmos-sdk/blob/v0.40.0/proto/ibc/applications/transfer/v1/tx.proto#L16-L19), I tend to believe this is overly explicit and `IbcMsg::Transfer` would be the better match for `ibc/MsgTransfer`.

Reply:
Happy to rename. Is there agreement on `IbcMsg::Transfer` ?